pom.xml

```diff
@@ -308,6 +307,7 @@
         <module>web-common</module>
         <module>servlet-dist</module>
         <module>servlet-feature-pack</module>
+        <module>feature-pack</module>
```

Comment:
I think you need to do ./build.sh install to install the servlet feature pack into the local repository.